sklearn/feature_extraction/hashing.py

```diff
@@ -150,23 +206,40 @@ def transform(self, raw_X):
             Feature matrix, for use with estimators or further transformers.
 
         """
-        raw_X = iter(raw_X)
-        if self.input_type == "dict":
-            raw_X = (_iteritems(d) for d in raw_X)
-        elif self.input_type == "string":
-            raw_X = (((f, 1) for f in x) for x in raw_X)
-        indices, indptr, values = \
-            _hashing.transform(raw_X, self.n_features, self.dtype,
-                               self.alternate_sign)
-        n_samples = indptr.shape[0] - 1
+        if self.save_mappings:
```

Comment:
No... We do not need this. we can check the feature map is stored and raise a error, but we still need transform to work, it just doesn't update the model